docs/book/emails.rst

```diff
@@ -7,73 +7,120 @@ E-Mails
 Sylius is sending various e-mails and this chapter is a reference about all of them. Continue reading to learn what e-mails are sent, when and how to customize the templates.
 To understand how e-mail sending works internally, please refer to :doc:`SyliusMailerBundle documentation </bundles/SyliusMailerBundle/index>`.
 
-User Confirmation E-Mail
-------------------------
+User Confirmation
+-----------------
 
-Every time new customer registers via registration form or checkout, ``user_confirmation`` e-mail is sent to him.
-The default template is
+Every time a new customer registers via the registration form, a user confirmation e-mail is sent to them.
 
-.. code-block:: text
+**Code**: ``user_confirmation``
 
-    SyliusWebBundle:Email:userConfirmation.html.twig
+**The default template**: ``SyliusCoreBundle:Email:userConfirmation.html.twig``
 
 You also have the following parameters available:
 
-user
-    Instance of the user model
+* ``user``: Instance of the user model
+
+Email Verification
+------------------
+
+When a new customer registers via the registration form, besides the User Confirmation an Email Verification is sent.
+
+**Code**: ``verification_token``
+
+**The default template**: ``SyliusCoreBundle:Email:verification.html.twig``
+
+You also have the following parameters available:
+
+* ``user``: Instance of the user model
+
+Password Reset
+--------------
+
+This e-mail is used when the user requests to reset their password in the login form.
+
+**Code**: ``reset_password_token``
+
+**The default template**: ``SyliusCoreBundle:Email:passwordReset.html.twig``
+
+You also have the following parameters available:
+
+* ``user``: Instance of the user model
 
 Order Confirmation
 ------------------
 
-This e-mail is sent when order is paid. Unique code is ``order_confirmation``. Template name is:
+This e-mail is sent when order is paid.
 
-.. code-block:: text
+**Code**: ``order_confirmation``
 
-    SyliusWebBundle:Email:orderConfirmation.html.twig
+**The default template**: ``SyliusCoreBundle:Email:orderConfirmation.html.twig``
 
 You also have the following parameters available:
 
-order
-    Instance of the user order
-order.user
-    Customer
-order.shippingAddress
-    Shipping address
-order.billingAddress
-    Billing address
-order.items
-    Collection of order items
+* ``order``: Instance of the order, with all its data
```

Comment:
hahaha wow, i am not a native English speaker myself so I slipped up too there 😋. So easy to make those.